### DISCUSSION TITLE
Allow manual DNS challange

### DISCUSSION THREAD

**Discussion by ckoeber83** [NginxProxyManager/nginx-proxy-manager#3614] [Ideas]
Hi,
I want to have manual DNS challange with TXT records on own Webspace to verify Wildcard domains.

Please add it to the DNS challange dialog:

![grafik](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/52466072/de7a5bc4-3e16-4299-ba20-b847b066f45e)

Thanks in advance!

**Comment by ohjohnsen** [comment]
There's already an open issue on this, sadly without much traction. 😢
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/813
